mycroft/util/lang/common_data_nl.py

```diff
@@ -0,0 +1,245 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Mycroft AI Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from collections import OrderedDict
+
+
+_ARTICLES = {'de', 'het'}
+
+
+_NUM_STRING_NL = {
+    0: 'nul',
+    1: 'een',
+    2: 'twee',
+    3: 'drie',
+    4: 'vier',
+    5: 'vijf',
+    6: 'zes',
+    7: 'zeven',
+    8: 'acht',
+    9: 'negen',
+    10: 'tien',
+    11: 'elf',
+    12: 'twaalf',
+    13: 'dertien',
+    14: 'veertien',
+    15: 'vijftien',
+    16: 'zestien',
+    17: 'zeventien',
+    18: 'achttien',
+    19: 'negentien',
+    20: 'twintig',
+    30: 'dertig',
+    40: 'veertig',
+    50: 'vijftig',
+    60: 'zestig',
+    70: 'zeventig',
+    80: 'tachtig',
+    90: 'negentig'
+}
+
+
+_FRACTION_STRING_NL = {
+    2: 'half',
+    3: 'derde',
+    4: 'vierde',
+    5: 'vijfde',
+    6: 'zesde',
+    7: 'zevende',
+    8: 'achtste',
+    9: 'negende',
+    10: 'tiende',
+    11: 'elfde',
+    12: 'twaalfde',
+    13: 'dertiende',
+    14: 'veertiende',
+    15: 'vijftiende',
+    16: 'zestiende',
+    17: 'zeventiende',
+    18: 'achttiende',
+    19: 'negentiende',
+    20: 'twintigste'
+}
+
+
+_LONG_SCALE_NL = OrderedDict([
+    (100, 'honderd'),
+    (1000, 'duizend'),
+    (1000000, 'miljoen'),
+    (1e12, "biljoen"),
+    (1e18, 'triljoen'),
+    (1e24, "quadriljoen"),
+    (1e30, "quintillion"),
+    (1e36, "sextillion"),
+    (1e42, "septillion"),
+    (1e48, "octillion"),
+    (1e54, "nonillion"),
+    (1e60, "decillion"),
+    (1e66, "undecillion"),
+    (1e72, "duodecillion"),
+    (1e78, "tredecillion"),
+    (1e84, "quattuordecillion"),
+    (1e90, "quinquadecillion"),
+    (1e96, "sedecillion"),
+    (1e102, "septendecillion"),
+    (1e108, "octodecillion"),
+    (1e114, "novendecillion"),
+    (1e120, "vigintillion"),
+    (1e306, "unquinquagintillion"),
+    (1e312, "duoquinquagintillion"),
+    (1e336, "sesquinquagintillion"),
+    (1e366, "unsexagintillion")
+])
+
+
+_SHORT_SCALE_NL = OrderedDict([
+    (100, 'hundred'),
```

Comment:
100 till 1e15 are not yet translated. Everything ening at *joen I consider fine, hardly used anyhow and about 99,99% of the Dutch people don't know them any way..